### PR TITLE
Add text to loading state cluster details

### PR DESCRIPTION
**Pull request by EMaksy**
# Description
This pull request updates the CheckResultsOverview component to implement the design provided by @jagabomb. As part of this update, the Spinner component was enhanced to improve customizability and accessibility.

### Before
beforeChangeDemo

### Now:
afterChangeDemo

## How was this tested?

Test was added for Spinner  component
Story was added for Spinner component

### CODE REVIEW

**Comment by jagabomb** [comment]
@EMaksy Thanks for this. A last request is to make the text gray for, **Check execution running...** Other than that it looks good to me.

File: assets/js/components/Spinner.jsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-import { computedIconCssClass } from '@lib/icon';
-
 import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
 
-function Spinner({ centered = false }) {
+function Spinner({
+  centered = false,
+  wrapperClassName = '',
+  size = 'm',
+  spinnerColor = 'fill-jungle-green-500',
```
Comment (nelsonkopliku) [inline assets/js/components/Spinner.jsx:8]: When do we need a custom color? :thinking:

Comment (dottorblaster) [reply assets/js/components/Spinner.jsx:8]: Agree

File: assets/js/components/Spinner.jsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
 import React from 'react';
-import { computedIconCssClass } from '@lib/icon';
-
 import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
 
-function Spinner({ centered = false }) {
+function Spinner({
+  centered = false,
+  wrapperClassName = '',
```
Comment (dottorblaster) [inline assets/js/components/Spinner.jsx:6]: this can be just `className`

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a thing then we can merge :shipit:

File: assets/js/components/Spinner.jsx (hunk 2 above)
Comment (EMaksy) [reply assets/js/components/Spinner.jsx]: Yes and also i don't think we need centered any more :D 
I checked out, it's not used in the codebase or did i miss something?

File: assets/js/components/Spinner.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+export const Default = {
+  argTypes: {
+    className: {
+      control: 'text',
+      description: 'Add padding or margin',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '""' },
```
Comment (nelsonkopliku) [inline assets/js/components/Spinner.stories.jsx:15]: what happens with `{ summary: '' },`?

**Review by nelsonkopliku — APPROVED** [review]

File: assets/js/components/Spinner.stories.jsx (hunk 3 above)
Comment (EMaksy) [reply assets/js/components/Spinner.stories.jsx:15]: Ah well done, nice catch. This is a left over! 
Storybook detects the default value. No need to add it here.

**Pull request merged**